test/integration/test-amphtml-ads.js

```diff
@@ -22,43 +22,46 @@ import {xhrServiceForTesting} from '../../src/service/xhr-impl';
 // TODO(wg-ads, #29112): Unskip on Safari.
 const t = describe.configure().skipSafari();
 
-t.run('AMPHTML ad on AMP Page', () => {
-  describes.integration(
-    'ATF',
-    {
-      amp: true,
-      extensions: ['amp-ad'],
-      body: `
+['a4a-no-signing:21066324', 'a4a-no-signing:21066325'].forEach((branch) => {
```

Comment:
can we reference to the IDs: NO_SIGNING_EXP.control

Reply:
yes, done.